rules/64-btrfs.rules.in

```diff
@@ -3,6 +3,7 @@
 SUBSYSTEM!="block", GOTO="btrfs_end"
 ACTION=="remove", GOTO="btrfs_end"
 ENV{ID_FS_TYPE}!="btrfs", GOTO="btrfs_end"
+ENV{SYSTEMD_READY}=="0", GOTO="btrfs_end"
```

Comment:
where does this come from?

Reply:
I couldn't find it in sysd either, and I'm not aware of anything using it, which is why I ask

Reply:
https://github.com/systemd/systemd/blob/main/rules.d/64-btrfs.rules.in#L6

Reply:
ah, huh, that slipped by my grepping. I am guessing this option has no impact on boot with btrfs? namely, I'm not sure whether that check fails or passes with SYSTEMD_READY unset